src/plugins/data/common/index_patterns/index_patterns/format_hit.ts

```diff
@@ -42,15 +42,16 @@ export function formatHitProvider(indexPattern: IndexPattern, defaultFormat: any
     hit: Record<string, any>,
     val: any,
     fieldName: string,
-    type: FieldFormatsContentType = 'html'
+    type: FieldFormatsContentType
   ) {
     const field = indexPattern.fields.getByName(fieldName);
     const format = field ? indexPattern.getFormatterForField(field) : defaultFormat;
 
     return format.convert(val, type, { field, hit, indexPattern });
   }
```

Comment:
This function `convert` is only used by this file, and since the only two usage of it passes `type` as an argument now, there is no more need to make this an optional param